### DISCUSSION TITLE
how to set conditional statement in index.template.html

### DISCUSSION THREAD

**Discussion by zeynab-rs** [quasarframework/quasar#9044] [General - Components / Directives / etc]
I want to insert a metadata only in one page of website. How to set conditional statement in index.template.html? This doesn't work:

```
<% if ( window.location.pathname == '/example' ) { %>
    <meta property="og:image" content="https://ogp.me/logo.png" />
<% } %>
```

**Comment by hawkeye64** [comment]
@heyran That because the conditionals in that template file become your regular index.html, so it happens during pre-processing, not at run time.

**Comment by ontwikkelfabriek** [comment]
@heyran 
You could use this plugin to change it for a single component/page:
https://quasar.dev/quasar-plugins/meta#how-it-works
